kolibri/core/device/migrations/0006_migrate_guest_access.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.core.exceptions import ObjectDoesNotExist
+from django.db import migrations
+from django.db import models
+from django.db.migrations.recorder import MigrationRecorder
+
+
+def migrate_allow_guest_access(apps, schema_editor):
+    # if this migration has been run, drop out because logic depends on a field that no longer exists
+    if MigrationRecorder.Migration.objects.filter(
+        app="kolibriauth",
+        name="0017_remove_facilitydataset_allow_guest_access",
+        applied__isnull=False,
```

Comment:
Why `__isnull=False`? Could it be better with a `.exclude(applied=True)` ?

Reply:
`applied` is actually a datetime field. It is not null when the migration has been applied.

Reply:
Ah of course! Thanks!